widgetssdk/build.gradle

```diff
@@ -7,8 +7,8 @@ android {
     defaultConfig {
         minSdkVersion 24
         targetSdkVersion 30
-        versionCode 12
-        versionName "1.6.7"
+        versionCode 13
+        versionName "1.7.8"
```

Comment:
looks like a typo, should be **1.6.8** or did I miss something?

Reply:
attributes count will change - it would be better to update to 1.7

Id replace 8 there to be same as versionCode, what do you think?

Reply:
version code set to 1.6.8